tests/test_code_gen.py

```diff
@@ -19,85 +18,100 @@
 
 import astor
 
+try:
+    from astunparse_common import AstunparseCommonTestCase
```

Comment:
astunparse is a third-party tool, right? If so, it should go into a separate test file. It would be better to separate core astor tests and third-party tool tests.